backend/private-graph/graph/schema.resolvers.go

```diff
@@ -2,6 +2,7 @@ package graph
 
 // This file will be automatically regenerated based on the schema, any resolver implementations
 // will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.24
```

Comment:
why this comment? this is making github hide the diff, and this file isn't generated

Reply:
This was autogenerated by make private-gen

Reply:
hmm but now the files are hidden in the diff tab. could we have this not generate or otherwise tell github to show the diffs?

![Screenshot 2023-02-06 at 1 40 44 PM](https://user-images.githubusercontent.com/1351531/217094169-e0ae0534-15bb-4005-be18-710dcde45d6a.png)

Reply:
hmm... looks like this is happening in main anyways without this change.